src/olympia/lib/akismet/models.py

```diff
@@ -80,6 +88,12 @@ def _post(self, action):
             url, data=self._get_data(), headers=self.HEADERS,
             timeout=settings.AKISMET_API_TIMEOUT)
 
+    def _statsd_incr(self):
+        metric = self.METRIC_LOOKUP.get(self.result, '')
+        statsd.incr(
+            'services.akismet.comment_check.%s.%s' % (
+                self.comment_type, metric))
```

Comment:
Yet another candidate where #955 would come in handy :cry:

Reply:
It would.  Looks like it's close to being usable to us.